Mathlib/Analysis/NormedSpace/Star/ContinuousFunctionalCalculus/Instances.lean

```diff
@@ -211,8 +211,8 @@ lemma spectrum_star_mul_self_nonneg {b : A} : ∀ x ∈ spectrum ℝ (star b * b
       simp [sup_eq_right.mpr hx.le]
   have h_c_spec₀ : SpectrumRestricts (- (star c * c)) (ContinuousMap.realToNNReal ·) := by
     simp only [SpectrumRestricts.nnreal_iff, h_eq_a_neg]
-    rw [← cfc_pow _ _ _ (.star_mul_self b)]
-    simp only [cfc_map_spectrum (R := ℝ) (star b * b) (fun x => (-ContinuousMap.id ℝ ⊔ 0) x ^ 3)]
+    rw [← cfc_pow _ _ (ha := .star_mul_self b)]
```

Comment:
```suggestion
    rw [← cfc_pow (ha := .star_mul_self b)]
```
Untested, but surely you don't need the underscores when you specify the arguments like this, no?

Reply:
Actually you *do*, at least for the first argument (`f`). This is because you need to force Lean to unify the argument at the appropriate time so that the auto-param for it runs.

Reply:
Oh dear! Alright then!